frontend/src/app/containers/App.js

```diff
@@ -100,14 +104,51 @@ class App extends Component {
       }
     });
     this.measurePageview();
+
+    const langs = {};
+
+    function addLang(lang, response) {
+      if (response.ok) {
+        response.text().then(data => {
+          langs[lang] = `${langs[lang] || ''}${data}
+`;
+        });
+      }
+    }
+
+    Promise.all(navigator.languages.map(language =>
```

Comment:
`navigator.languages` might include languages that Test Pilot doesn't support. You can use [fluent-langneg](https://github.com/projectfluent/fluent.js/tree/master/fluent-langneg) to make sure you only fetch files that are known to exist.